make-all.sh

```diff
@@ -4,6 +4,8 @@ RED='\033[0;31m'
 GREEN='\033[0;32m'
 NC='\033[0m'
 
+IMAGE_NAME=jellyninjas/terminalfighterdocs
```

Comment:
I think using the caps I suggested would be better. Normal naming for docker containers.

Reply:
Caps don't work as Docker names